cni-plugins/sriov-passthrough-cni/README.md

```diff
@@ -27,3 +27,37 @@ plugin on all the relevant K8s nodes.
 │   └── sriov-passthrough-cni
 └── README.md
 ```
+
+## Prow job usage:
+
+### In order to use the CNI and allocate one sriov PF:
+1. Add the following annotation to the prow job yaml (or leave the current annotation unchanged).
+
+`k8s.v1.cni.cncf.io/networks: '[{"interface":"net1","name":"sriov-passthrough-cni","namespace":"multus-cni-ns"}`
+
+2. When the parallel sriov jobs mode is enabled, add the following requests/limits:
+```
+requests:
+  prow/sriov: "1"
+limits:
+  prow/sriov: "1"
+```
+
+### In order to run the CNI twice, so 2 sriov PFs would be allocated per job:
+1. Use the following annotation (added 2nd entry with just a different name for the interface),
+must be in the same line, else the line will be deduped, as it has the same key.
+
+`k8s.v1.cni.cncf.io/networks: '[{"interface":"net1","name":"sriov-passthrough-cni","namespace":"multus-cni-ns"}, {"interface":"net2","name":"sriov-passthrough-cni","namespace":"multus-cni-ns"}]'`
+
+2. When the parallel sriov jobs mode is enabled, add the following requests/limits:
+update requests/limits accordingly, so the scheduler would preserve 2 prow/sriov resources for each job,
+and obey the prow/sriov limit per node.
+```
+requests:
+  prow/sriov: "2"
+limits:
+  prow/sriov: "2"
+```
+
+Note:
+`sriov-pod: "true"` can be used as a safety catch if sriov jobs sequential mode is desired.
```

Comment:
What do you mean?

Can you describe what exactly "safety catch" is and how to use `sriov-pod: "true"` as safety catch ?

Reply:
Removed it, its too much details, and not related the what the README should explain.